instrumentation/grape/README.md

```diff
@@ -30,6 +41,32 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+### Configuration options
+
+#### `:ignored_events` (array)
+
+Indicate if any events should not produce spans.
+
+- Accepted values: `:endpoint_render`, `:endpoint_run_filters`.
+- Defaults to `[]` (no ignored events).
+
+Example:
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::Grape', { ignored_events: [:endpoint_run_filters] }
+end
+```
+
+Note that the `endpoint_run` event cannot be disabled since it is the parent event. If you need to disable the instrumentation, set `:enabled` to `false`:
```

Comment:
I don't think I agree with that reasoning.

Reply:
I would like to still exclude `endpoint_run` from the "ignorable" events, if you're ok with that, for the reason mentioned in [this comment](https://github.com/catawiki/opentelemetry-ruby-contrib/pull/3#discussion_r1144347132). It is indeed possible to exclude it, but I think that if we allowed that, we should add a big warning to the README explaining that if you ignore `endpoint_run`, all the other events will now be children of a span produced by another middleware, and I would like to keep it as simple as possible (at least in this first implementation).

I have made the change Ben suggested to the README to remove its being the parent as the cause. Now it just says that it cannot be ignored and that's it 😅